packages/calcite-components/src/components/alert/alert.e2e.ts

```diff
@@ -11,6 +11,10 @@ describe("defaults", () => {
       propertyName: "autoCloseDuration",
       defaultValue: "medium",
     },
+    {
+      propertyName: "embedded",
```

Comment:
We should avoid testing internal props unless there's a strong use case.

Reply:
I know it's an internal prop, but we should still test it to expect it to be false by default IMO. This adds protection from someone removing it accidentally or its value being changed.